Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -124,9 +129,21 @@ struct SettingsView: View {
                             } label: {
                                 Text("Delete", comment: "Label to delete a vehicle")
                             }
+                            Button {
+                                isEditingVehicle = true
```

Comment:
Here is where you would put `vehicleToEdit = vehicle`, because this is in the `ForEach` loop, and it's specifying which vehicle is going to show in the `EditVehicleView`